openerp/fields.py

```diff
@@ -1504,6 +1624,17 @@ def __init__(self, selection=Default, string=Default, **kwargs):
             selection = api.expected(api.model, selection)
         super(Selection, self).__init__(selection=selection, string=string, **kwargs)
 
+    @property
+    def column_type(self):
+        if self.size == -1 or (self.selection and
```

Comment:
It was use when the selection was get via a callable that return a name_get of records...
m2o with widget=selection are used now.